src/main/java/ParkingLot/Admin.java

```diff
@@ -0,0 +1,20 @@
+package ParkingLot;
+
+public class Admin extends Employees {
```

Comment:
Employees is good name for List<Employee> variable, not for a class

Reply:
fixed